lang/algol68g/files/0001-configure.ac-fix-triples.patch

```diff
@@ -38,15 +38,6 @@ index 6bd1d2f..74d5586 100644
    AC_DEFINE(BUILD_BSD, 1, [Define this if NetBSD was detected]) 
    AC_DEFINE(HAVE_IEEE_754, 1, [Define this if IEEE_754 compliant]) 
    AC_MSG_RESULT([netbsd])
-@@ -99,7 +99,7 @@ case "$host" in
- # OpenBSD.
- #
- 
--*86-*-openbsd* | *86_64-*-openbsd*) 
-+*-*-openbsd*) 
-   AC_DEFINE(BUILD_BSD, 1, [Define this if OpenBSD was detected]) 
-   AC_DEFINE(HAVE_IEEE_754, 1, [Define this if IEEE_754 compliant]) 
-   bsd_include_dirs=yes
 @@ -259,7 +259,7 @@ else
```

Comment:
Why is removing OpenBSD support necessary to making this build?

Reply:
We are not removing anything, that was finally set correctly by upstream. So we do not need to fix it in the patch. (It failed to apply in the earlier form.)